.github/workflows/deploy-prod.yaml

```diff
@@ -0,0 +1,63 @@
+name: Deploy Prod
+
+on:
+  release:
+    types: [published, created, edited]
```

Comment:
do we need to deploy to Prod on an edited release?

Reply:
Yes, if someone edits a release and mark it as latest it should get triggered